.circleci/config.yml

```diff
@@ -179,6 +178,9 @@ jobs:
     parallelism: 4
     steps:
       - attach_workspace: *attach_options
+      - run:
+          name: Initialize Environment
+          command: ./.circleci/env.sh
```

Comment:
If we already have a env script for Linux, should we move the `~/.npm-global/bin` stuff there too?

Reply:
I was actually considering doing the opposite long-term.  `environment` is cross-platform whereas the shell scripts are not.

Reply:
Yes, that makes more sense.